src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.targets

```diff
@@ -99,7 +99,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Controls whether or not the static web assets feature is enabled. By default is enabled for netcoreapp3.0
          applications and RazorLangVersion 3 or above. -->
     <StaticWebAssetsEnabled Condition="'$(StaticWebAssetsEnabled)' == ''">$(_Targeting30OrNewerRazorLangVersion)</StaticWebAssetsEnabled>
-    <UseStaticWebAssetsV1 Condition="'$(UseStaticWebAssetsV1)' == '' And ('$(_TargetingNET60OrLater)' == 'true' Or '$(_TargetingNETStandard21OrLater)' == 'true')">false</UseStaticWebAssetsV1>
+    <UseStaticWebAssetsV1 Condition="'$(UseStaticWebAssetsV1)' == '' And ('$(_TargetingNET60OrLater)' == 'true' Or '$(_TargetingNETStandard21OrLater)' == 'true') And '$(RazorLangVersion)' > '3.0'">false</UseStaticWebAssetsV1>
```

Comment:
Why are netstandard and 6.0 projects considered equivalent? Can we simply use net6.0 as a way to determine if it needs to use the v2 version of the manifest?

Reply:
blazor webassembly 3.1 is netstandard 2.1 and needs to use the old targets. However, I want netstandard2.1 libraries to use the new static web assets so that existing RCLs produce the assets in the new package format.

The lang requirement discriminates between old blazor webassembly apps and regular RCLs

Reply:
> However, I want netstandard2.1 libraries to use the new static web assets 

If the two formats are entirely compatible, can we leave it be? It seems problematic because the users may have different experiences depending on what SDK they are using to build their app.

Reply:
Also, doesn't this affect legacy apps? Using a ns2.1 library with a 3.1 or 5.0 app is a supported scenario.

Reply:
I'm changing the logic here on your suggestion to just use 6.0 and onwards.
For now, we can require rcls that target a previous version to opt-in with the flag.
And in RC1 I have an idea on how to make this backwards compatible.